.github/workflows/ensureSCCoreDevApproval.yml

```diff
@@ -59,11 +61,15 @@ jobs:
 
           echo -e "$MEMBERS" > sc_core_dev_members.txt
           echo "CONTINUE=true" >> $GITHUB_ENV
-
+      ## Get the PR Number even if its just a push event
```

Comment:
I think it's not really necessary to run it on every push event, is it? 
That was just there for testing.
Should be sufficient to just run this on pull_request events.